### DISCUSSION TITLE
Developers check why the file is displayed with a virus

### DISCUSSION THREAD

**Discussion by mk2607** [Drakkar-Software/OctoBot#2464] [General]
Hello! Dear developers, please see why octobot for windows is defined by Kaspersky as a virus? And unlock me in telegram
![окто](https://github.com/Drakkar-Software/OctoBot/assets/32495968/728ff42e-5adb-4705-9e25-1d0698829895)

**Comment by GuillaumeDSM** [comment]
Hey @mk2607, we recently get a lot of spammer chilling scams on telegram, it's possible that we mistakenly banned you, sorry for that. Please pm me your telegram handle for me to unban you, this was not intended. My telegram handle is `@kylemon`
As you can see, the whole OctoBot project is open source and you can check the code yourself if you are afraid that it might contain viruses.
Another user on discord also reported that Kaspersky is flagging the executable as "malicious". We will investigate the issue to understand what is really going on there and we come back to you.

**Reply by mk2607** [reply]
Thanks!

**Comment by GuillaumeDSM** [comment]
Update on the situation:
We reached out to the Kaspersky technical support for a double check and they just confirmed that it's a false positive. 
=> There is no virus in the software.

OctoBot executable and download links will be authorized again in the next version of Kaspersky.

Thank you again @mk2607  for notifying us about this issue.

**Reply by mk2607** [reply]
Hi, I just downloaded the latest version of octobot for windows, Kaspersky really fixed the problem with a false result and now the file is downloaded without problems and is not recognized as a virus! Thank you for your work, friends! Develop only for the better!

**Reply by GuillaumeDSM** [reply]
Great ! Thank you for coming back to us on this !
